shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/BaseTypes.hs

```diff
@@ -186,7 +188,7 @@ deriving anyclass instance FromJSON Nonce
 
 -- | Evolve the nonce
 (⭒) :: Nonce -> Nonce -> Nonce
-(Nonce a) ⭒ (Nonce b) = Nonce . coerce $ hash @SHA256 (getHash a <> getHash b)
+(Nonce a) ⭒ (Nonce b) = Nonce . coerce $ hashRaw @SHA256 id (getHash a <> getHash b)
```

Comment:
This no longer hashes the CBOR bytestring prefix

Reply:
:clap: